boards/common/xsdb.board.cmake

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+board_set_runner(FLASH xsdb)
```

Comment:
Why is this line needed? None of the other *.board.cmake files seem to set this.